packages/appstore/src/components/cfds-listing/index.tsx

```diff
@@ -140,14 +140,24 @@ const CFDsListing = () => {
                         const has_mt5_account_status = existing_account.status
                             ? getMT5AccountAuthStatus(existing_account.status)
                             : null;
+                        const is_demo_account = existing_account.account_type === 'demo';
```

Comment:
We already have is_demo to check for demo accounts, what is the reason of defining it again ?

Reply:
Done